src/control/control/nodes/test_path.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+from nav_msgs.msg import Odometry
+import rclpy
+from rclpy.node import Node
+
+class TestPath(Node):
+
+    def __init__(self):
+        super().__init__('test_path')
+        self.path_publisher = self.create_publisher(Odometry, 'path', 10)
```

Comment:
Change `path` to `generated_path`. Also the path publisher should have format `GeneratedPath` not `Odometry`

Reply:
I believe we discussed this at the meeting. If not, I believe that it is best to integrate path following as a separate PR.